org.kryogenix.Pick.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.kryogenix.Pick
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: pick-colour-picker
+
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
```

Comment:
it doesn't support wayland? If you use the GNOME SDK I'll assume it's because it uses pygtk, which mean that it's gtk3 or 4, so wayland.

Reply:
It picks colours off the screen. As I understand it, this doesn’t work in wayland and can’t be done unless you’re the shell, so Pick doesn’t do Wayland. If that’s incorrect and gtk glosses over the difference then I’m happy to look at supporting wayland in a future release.

Reply:
nah that's a reason good enough,